src/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Userprofile from './components/Userprofile.jsx';
```

Comment:
Great job @BukkyOmo on the environment setup. I think we should still stick to the file naming convention as in `User-profile.jsx`

Reply:
Fixed.